appveyor.yml

```diff
@@ -1,6 +1,16 @@
+clone_folder: c:\go\src\github.com\github\git-lfs
+
+environment:
+  GOPATH: c:\go
+
 install:
+  - set PATH=%GOPATH%\bin;%PATH%;
+  - go version
   - cinst InnoSetup -y
-  - set PATH="C:\Program Files (x86)\Inno Setup 5";%PATH%
+  - set PATH="C:\Program Files (x86)\Inno Setup 5";%PATH%;
+
+test_script:
```

Comment:
How about moving this block after `after_build`? I feel that would be more natural as it matches execution order.